src/test/java/com/softserveinc/ita/petrus/TestRunner.java

```diff
@@ -0,0 +1,29 @@
+package com.softserveinc.ita.petrus;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class TestRunner {
+    WebDriver driver;
+
+    @BeforeClass
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/driver/chromedriver.exe");
```

Comment:
add to config property, but create two paths, one for windows path (such as yours), and one for UNIX "src//main//resources//driver//chromedriver" and in properties util class implement condition depending on the operating system and return path for specific OS